### PR TITLE
gretty fix

### PR DESCRIPTION
**Pull request by akhikhl**

### CODE REVIEW

File: ch02/groovybaseball/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'org.akhikhl.gretty:gretty:2.0.0'
+        classpath 'org.gretty:gretty:2.1.0'
```
Comment (akhikhl) [inline ch02/groovybaseball/build.gradle]: Gretty is now community-driven, it's coordinates has changed.

File: ch02/groovybaseball/build.gradle [hunk 2]
```diff
@@ -16,7 +16,7 @@ plugins {
 }
 
 
-apply plugin: 'org.akhikhl.gretty'
+apply plugin: 'org.gretty'
```
Comment (akhikhl) [inline ch02/groovybaseball/build.gradle]: pluginId for community-driven Gretty is now different.

File: ch02/groovybaseball/build.gradle [hunk 3]
```diff
@@ -57,7 +57,7 @@ task wrapper(type:Wrapper) {
 }
 
 dependencies {
-    compile 'org.codehaus.groovy:groovy-all:2.4.14'
+    compile 'org.codehaus.groovy:groovy-all:2.4.12'
```
Comment (akhikhl) [inline ch02/groovybaseball/build.gradle]: There seems to be incompatibility between Jetty annotation parser and groovy:2.4.14. My quick fix is to use older version of groovy.

Comment (kousen) [reply ch02/groovybaseball/build.gradle]: Yes, this fixed the problem. Thanks! Hopefully you can do a longer term fix later.

**Pull request merged**